internal/x/phab/phab.go

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) 2018 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+// Package phab provides functionality to interact with Phabricator.
+//
+// The primary purpose is to convert failures to JSON that is compatible
+// with the Harbormaster API.
+//
+// https://secure.phabricator.com/conduit/method/harbormaster.sendmessage
+package phab
+
+import "github.com/uber/prototool/internal/x/text"
+
+const (
```

Comment:
Should we un-export these?

Reply:
I think they should be constant and in the GoDoc (as other packages in Prototool do for now). Note everything will stay internal as it is. But just my opinion.

Reply:
I'm in favour of keeping them public for godocs since this is in an `internal` package. This allows users to see how we're handling Harbormaster output without trial/error.

Reply:
I’d vote for this too. Note that this can change later - let’s keep it in x/, the goal here really is to make sure we’re all “generally” on the same page so I can trial this internally.

@amckinney is this OK? Up to you two really.

Reply:
Fine with me! Just thought it was worth mentioning. This makes sense with respect to a godoc perspective.